src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -28,7 +28,7 @@
 
 public class IntegrityCheck {
 
-    private final BibDatabaseContext bibDatabaseContext;
+    private static BibDatabaseContext bibDatabaseContext;
```

Comment:
Many FieldChecker need access to the isBiblatexMode() method. It is bad to create for every FieldChecker class a new bibDatabaseContext object and inizialize it (in the FieldChecker constructor?).